packaging/cloudify-logstash.spec

```diff
@@ -0,0 +1,99 @@
+
+Name:           cloudify-logstash
+Version:        %{CLOUDIFY_VERSION}
+Release:        %{CLOUDIFY_PACKAGE_RELEASE}%{?dist}
+Summary:        Cloudify's Logstash
+Group:          Applications/Multimedia
+License:        Apache 2.0
+URL:            https://github.com/cloudify-cosmo/cloudify-amqp-influxdb
+Vendor:         Cloudify Platform Ltd.
+Packager:       Cloudify Platform Ltd.
+
+BuildRequires:  jre, rsync, logstash = 1:1.5.0
+Requires:       jre, postgresql94-jdbc
+Conflicts:      logstash
+
+Source0:        http://repository.cloudifysource.org/cloudify/components/logstash-output-jdbc-0.2.10.gem
+Source1:        http://repository.cloudifysource.org/cloudify/components/logstash-filter-json_encode-0.1.5.gem
+
+%define _user logstash
+
+# Work around lurking stale hashbangs from logstash's build process
+%define __requires_exclude ^/home/jenkins/.*
+
+
+%description
+Cloudify's logstash plugins and configuration
+
+
+%install
+
+/opt/logstash/bin/plugin install ${RPM_SOURCE_DIR}/*.gem
+
+
+# Copy entire logstash install
+mkdir %{buildroot}/opt
+cp -R /opt/logstash %{buildroot}/opt/logstash
+
+# move & edit init script
+mkdir -p %{buildroot}/etc/init.d
+cp /etc/init.d/logstash %{buildroot}/etc/init.d
+sed -i 's:sysconfig/\$name:sysconfig/cloudify-$name:g' %{buildroot}/etc/init.d/logstash
+
+# Create the log dir
+mkdir -p %{buildroot}/var/log/cloudify/%_user
+
+# Create var dir
+mkdir -p %{buildroot}/var/lib/logstash
+
+# Copy static files into place. In order to have files in /packaging/files
+# actually included in the RPM, they must have an entry in the %files
+# section of this spec file.
+cp -R ${RPM_SOURCE_DIR}/packaging/logstash/files/* %{buildroot}
+
+
+%pre
+
+# create logstash group
+if ! getent group logstash >/dev/null; then
```

Comment:
eh if we have %_user maybe w can have %_group as well

Reply:
These sections are copied from the logstash RPM, because we're bundling the whole application. Hopefully we can drop logstash rather than needing to upgrade it, but in case things need to be merged again I'd rather make minimal changes in copied sections